src/KM_Controls.pas

```diff
@@ -2491,8 +2612,8 @@ function TKMEdit.KeyEventHandled(Key: Word; Shift: TShiftState): Boolean;
     VK_END: Result := (fText <> ''); //These keys have no effect when text is blank
   end;
 
-  //We want these keys to be ignored by chat, so game shortcuts still work
-  if Key in [VK_F1..VK_F12, VK_ESCAPE] then Result := False;
+  //We want these keys to be ignored by TKMEdit
+  if Key in [VK_F1..VK_F12, VK_ESCAPE, VK_RETURN, VK_TAB] then Result := False;

```

Comment:
yeah, you can send with return key ofc, because it is handled in KM_GUIGameChat unit. I deleted this comment because we have many TKEdits, which hasnothing with chat, so its correct to point it here.
But what is a bit wrong with this code, that we need to return True from TKMEdit.KeyDown method, when Enter pressed. This is result should be passed from KM_GUIMessage, because this is the place, where return key handled, and other TKMEdit may not handle it. 
I will do some refactoring due to this, Thanks!